### PR TITLE
Improvements in JNetReflector and use Java types instead of Nullable .NET native types

### PR DESCRIPTION
**Pull request by masesdevelopers**
## Description

The PR updates:
- JNetReflector with:
  - Added **UseDotNetNullable** switch if the user wants .NET nullable types, by default it use Java types
  - Added **ClassesManuallyDeveloped** to list classes will be manually developed
- JNet: code regenerated

## How Has This Been Tested?

To be tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/net/JNet/Developed/Java/Lang/Class.cs [hunk 1]
```diff
@@ -312,4 +314,38 @@ public class Class<T> : Class where T : IJVMBridgeBase
...
+        /// <typeparam name="T">The type implementing <see cref="IJVMBridgeBase"/></typeparam>
+        /// <returns>The <see cref="Class{T}"/> object for the class with the specified name.</returns>
+        public static Class<T> Of<T>() where T : IJVMBridgeBase, new()
+        {
+            return SExecute<Class<T>>("forName", ClassNameOf<T>());
+        }
+
+        public T GetAnnotation<T>(Class arg0) where T : IAnnotation, new()
+        {
+            throw new global::System.NotImplementedException();
```
Comment (mariomastrodicasa) [inline src/net/JNet/Developed/Java/Lang/Class.cs:332]: Implement the function

File: src/net/JNet/Developed/Java/Lang/Class.cs [hunk 2]
```diff
@@ -312,4 +314,38 @@ public class Class<T> : Class where T : IJVMBridgeBase
...
+        }
+
+        public T[] GetAnnotationsByType<T>(Class arg0) where T : IAnnotation, new()
+        {
+            throw new global::System.NotImplementedException();
```
Comment (mariomastrodicasa) [inline src/net/JNet/Developed/Java/Lang/Class.cs:337]: Implement the function

File: src/net/JNet/Developed/Java/Lang/Class.cs [hunk 3]
```diff
@@ -312,4 +314,38 @@ public class Class<T> : Class where T : IJVMBridgeBase
...
+        }
+
+        public T GetDeclaredAnnotation<T>(Class arg0) where T : IAnnotation, new()
+        {
+            throw new global::System.NotImplementedException();
```
Comment (mariomastrodicasa) [inline src/net/JNet/Developed/Java/Lang/Class.cs:342]: Implement the function

File: src/net/JNet/Developed/Java/Lang/Class.cs [hunk 4]
```diff
@@ -312,4 +314,38 @@ public class Class<T> : Class where T : IJVMBridgeBase
...
+        }
+
+        public T[] GetDeclaredAnnotationsByType<T>(Class arg0) where T : IAnnotation, new()
+        {
+            throw new global::System.NotImplementedException();
```
Comment (mariomastrodicasa) [inline src/net/JNet/Developed/Java/Lang/Class.cs:347]: Implement the function

**Review by mariomastrodicasa — COMMENTED** [review]
There are some missing implementations

File: src/net/JNet/Developed/Java/Lang/Class.cs (hunk 2 above)
Comment (masesdevelopers) [reply src/net/JNet/Developed/Java/Lang/Class.cs:337]: Will be done with #200

File: src/net/JNet/Developed/Java/Lang/Class.cs (hunk 3 above)
Comment (masesdevelopers) [reply src/net/JNet/Developed/Java/Lang/Class.cs:342]: Will be done with #200

File: src/net/JNet/Developed/Java/Lang/Class.cs (hunk 1 above)
Comment (masesdevelopers) [reply src/net/JNet/Developed/Java/Lang/Class.cs:332]: Will be done with #200

File: src/net/JNet/Developed/Java/Lang/Class.cs (hunk 4 above)
Comment (masesdevelopers) [reply src/net/JNet/Developed/Java/Lang/Class.cs:347]: Will be done with #200

**Pull request merged**